flake8_quotes/__init__.py

```diff
@@ -2,12 +2,17 @@
 import tokenize
 import warnings
 
-import pep8
+import flake8
 
 from flake8_quotes.__about__ import __version__
 
+FLAKE8_VERSION2 = int(flake8.__version__.split('.')[0]) == 2
+if FLAKE8_VERSION2:
+    # Use the pep8/pycodestyle package used by Flake8
+    from flake8.engine import pep8
```

Comment:
I added a print of `file_contents` in `run` and here is what you get:

```
$ echo 'f = "foobar"' | flake8 - --select=Q
[]
$ flake8 --version
3.0.0b2 (pycodestyle: 2.0.0, mccabe: 0.5.0, pyflakes: 1.2.3, flake8_quotes: 0.5.0, flake8-future-import: 0.4.3) CPython 3.5.1 on Linux
$ git rev-parse HEAD
26f7a56a3ca1af576609c97ca42de992bc92ef82
```

This is running the first commit of this PR (so without the two class-magic). I actually mention this in the OP:

> Additionally with Flake8 3.x it seems that it doesn't use pycodestyle's implementation at all anymore. Instead Flake8 itself first reads the content and caches it. But this now causes the fix to return an empty string.